### DISCUSSION TITLE
Does the loss of positive and negative active materials not cause loss of battery capacity? Weird.

### DISCUSSION THREAD

**Discussion by RobinBluehub** [pybamm-team/PyBaMM#2321] [Q&A] [answered]
Hi, pybamm. I found that the sum of the capacity loss from Li plating and SEI to battery capacity is exactly equal to the total battery capacity loss. And the LAM contribution is very small, why is this? Hope to answer.This is very inconsistent with the situation in OKane's doctoral dissertation.
![image](https://user-images.githubusercontent.com/103869258/192450736-c7c52c1f-f520-4559-8089-174d0d09b4e2.png)
![image](https://user-images.githubusercontent.com/103869258/192451097-3fd85d18-757c-4c33-b6f8-fe05a564d7e4.png)

**Comment by RobinBluehub** [comment]
Also I would like to ask when will the next version of pybamm be released?

**Reply by valentinsulzer** [reply]
At the end of the month

**Comment by valentinsulzer** [comment, marked as answer]
Lost capacity is not just a linear sum of LAM and LLI. The dependence is more complicated. See for example section 3 of https://doi.org/10.1016/j.jpowsour.2019.03.104
